airbyte-integrations/connectors/source-airtable/source_airtable/source.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+from airbyte_cdk.logger import AirbyteLogger
+from airbyte_cdk.models import AirbyteCatalog, AirbyteStream
+from airbyte_cdk.models.airbyte_protocol import DestinationSyncMode, SyncMode
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.requests_native_auth import TokenAuthenticator
+
+
+# Basic full refresh stream
+class AirtableStream(HttpStream, ABC):
+    url_base = "https://api.airtable.com/v0/"
+    primary_key = "id"
+
+    def __init__(self, base_id: str, table_name: str, schema, **kwargs):
+        super().__init__(**kwargs)
+        self.base_id = base_id
+        self.table_name = table_name
+        self.schema = schema
+
+    @property
+    def name(self):
+        return self.table_name
+
+    def get_json_schema(self) -> Mapping[str, Any]:
+        return self.schema
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        json_response = response.json()
+        offset = json_response.get("offset", None)
+        if offset:
+            return {"offset": offset}
+        return None
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        if next_page_token:
+            return next_page_token
+        return {}
+
+    def process_records(self, records):
+        processed_records = []
+        for record in records:
+            processed_record = {
+                "_airtable_id": record.get("id"),
+                "_airtable_created_time": record.get("createdTime"),
+            }
+            data = record.get("fields", {})
+            # Convert all values to string
+            data = {key: str(value) for key, value in data.items()}
```

Comment:
Please checkout our [Type Transformation](https://docs.airbyte.io/connector-development/cdk-python/schemas#type-transformation) documentation. You could delegate this kind of transformation to a `TypeTransformer`

Reply:
I had this before: `transformer = TypeTransformer(TransformConfig.CustomSchemaNormalization)`
![image](https://user-images.githubusercontent.com/52090179/141227119-8bcac6e1-0c15-4d17-adcb-8cbae5167c1c.png)
But still got this error running integration tests.